### DISCUSSION TITLE
Could the Day One output format convert @ to # in tags?

### DISCUSSION THREAD

**Discussion by jackbaty** [ttscoff/doing#165] [Ideas]
I just discovered the `-o dayone` formatter and it's so much nicer than the janky shell script I've been using.

One thing I'd love is if tags could be formatted using "#" instead of "@" when sending to Day One. Just a thought.

**Comment by ttscoff** [comment]
Tags should be applied as actual tags, rather than embedded. Are you not 
getting your entries tagged?

-Brett

**Reply by jackbaty** [reply]
Oh geez, I didn't even notice that was happening, sorry! It works great as-is. Carry on 😄
